src/weather.js

```diff
@@ -0,0 +1,10 @@
+class Weather {
+
+    getWeather() {
+        return Math.floor(Math.random() * 10 > 3) ? `clear` : `stormy`;
+    }
+
```

Comment:
This was the shortest, and most loosely coupled, way I was able to think of creating the getWeather() method, but it does not feel like the clearest or simplest way.